packages/SidePanel/SidePanel.stories.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Heading from "@paprika/Heading";
+
+import SidePanel from "./src";
+
+const SidePanelStory = () => {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const toggle = () => {
+    setIsOpen(state => !state);
+  };
+
+  return (
+    <React.Fragment>
+      hello lets open a sidepanel
+      <SidePanel isOpen={isOpen}>
+        <SidePanel.Trigger onClick={toggle}>{isOpen ? "close" : "open"}</SidePanel.Trigger>
+        <SidePanel.Overlay onClick={toggle} />
```

Comment:
I have some concerns about the `SidePanel.Overlay` component:
1. An overlay is visually displayed not inside a `SidePanel`, but around it. I think it's not so obvious for a consumer to place `SidePanel.Overlay` as a child of `SidePanel` component.
2. If a consumer should render `SidePanel.Overlay` component manually, it automatically becomes "not required". Although looking at Projects module, there is only one `SidePanel` without an overlay (new To-do SidePanel). All the other side panels have it. Also, I believe that almost always a side panel should be closed if a user clicks on an overlay. Now a consumer should specify it twice: `<SidePanel onClose={this.handleClose}>` and `<SidePanel.Overlay onClick={this.handleClose}`. Maybe it's better to display an overlay by default and call `onClose` on overlay click. Then props for overlays can be passed like so:
```js
<SidePanel isOpen={isOpen} showOverlay={false}>...</SidePanel>
<SidePanel isOpen={isOpen} overlayBackground={...}>...</SidePanel>
OR
<SidePanel isOpen={isOpen} overlay={{ show: false }}>...</SidePanel>
<SidePanel isOpen={isOpen} overlay={{ background: "..." }}>...</SidePanel>
```
WDYT?